custom_components/vicare/binary_sensor.py

```diff
@@ -27,14 +27,38 @@
 SENSOR_CIRCULATION_PUMP_ACTIVE = "circulationpump_active"
 SENSOR_BURNER_ACTIVE = "burner_active"
 SENSOR_COMPRESSOR_ACTIVE = "compressor_active"
-
+SENSOR_FROST_PROTECTION_ACTIVE = "frost_protection_active"
+SENSOR_DHW_CIRCULATION_PUMP_ACTIVE = "dhw_circulation_pump_active"
+SENSOR_CHARGING_ACTIVE = "charging_active"
+SENSOR_DHW_PUMP_ACTIVE = "dhw_pump_active"
 
 @dataclass
 class ViCareBinarySensorEntityDescription(
     BinarySensorEntityDescription, ViCareRequiredKeysMixin
 ):
     """Describes ViCare binary sensor entity."""
 
+GLOBAL_SENSORS: tuple[ViCareBinarySensorEntityDescription, ...] = (
+    ViCareBinarySensorEntityDescription(
+        key=SENSOR_DHW_CIRCULATION_PUMP_ACTIVE,
+        name="DHW Circulation Pump Active",
+        device_class=DEVICE_CLASS_POWER,
+        value_getter=lambda api: api.getDomesticHotWaterCirculationPumpActive(),
+    ),
+    ViCareBinarySensorEntityDescription(
+        key=SENSOR_CHARGING_ACTIVE,
+        name="Charging Active",
+        device_class=DEVICE_CLASS_POWER,
```

Comment:
I guess device_class  should be DEVICE_CLASS_RUNNING or similar?

Reply:
I've just looked at the full list of available device classes for binary sensors:
https://www.home-assistant.io/integrations/binary_sensor/
But I can't find how it will be represented in UI, where I could find full reference to all data classes?

Reply:
Just checked how it looks like:
![image](https://user-images.githubusercontent.com/19752098/145989187-93aa5d46-951b-4d04-b45f-98d3fdc43918.png)
I like more the original DEVICE_CLASS_POWER but I'm open :) It is anyway not going to be a key sensor with focused display.

Reply:
I guess we will learn what is the right device type once we contribute to HA Core ;)